src/apm-cli.js

```diff
@@ -116,10 +116,10 @@ const printVersions = function(args, callback) {
     let versions;
     if (args.json) {
       versions = {
-        apm: apmVersion,
+        ppm: apmVersion,
         npm: npmVersion,
         node: nodeVersion,
-        atom: atomVersion,
+        pulsar: atomVersion,
         python: pythonVersion,
         git: gitVersion,
         nodeArch: process.arch
```

Comment:
Hmm.

Wondering if I need to stuff `apm` and `atom` into the JSON output, since that is presumably meant for programmatic use. I'm admittedly not familiar with how this JSON output is used, but wouldn't be at all surprised if this is relevant to, say, checking if the atomVersion is a supported version in the package's `engines` filed in `package.json`?

Or possibly leave the JSON output not re-branded at all, leave the interface for programmatic use totally 100% unchanged. (Adding new properties seems safe-ish to me, deleting `atom` and `apm` might be disruptive either to core editor if it's used there (not sure) and probably has some impact when interacting with the package backend??)

Reply:
`./bin/ppm install` still works on this branch, though, as does `./bin/ppm install --json`. So, I haven't found a place where this clearly breaks stuff with just trying `ppm install` so far.

Reply:
You could do all of that, although if it were me, I'd opt to just adding new properties, if it is something you are worried about.